simpleflow/local/executor.py

```diff
@@ -53,6 +57,18 @@ def initialize_history(self, input):
             self._workflow_class,
             input=input)
 
+    def on_new_workflow(self, task):
+        self.wf_run_id.append(
+            "local_{}_{}".format(task.workflow.name.lower(), uuid.uuid4()),
```

Comment:
I think the uuid would be enough

Reply:
as seen with you this cannot be done yet.